tensorflow_hub/module_v2.py

```diff
@@ -44,10 +44,14 @@ def resolve(handle):
   Returns:
     A string representing the Module path.
   """
-  return registry.resolver(handle)
+  return registry.resolver(handle, bypass_cert)
 
+def _by_pass_cert_validation(bypass=False):
+  # TODO Document this and add more error logic
+  # bypass should be True or [False]
+  return bypass
 
-def load(handle, tags=None, options=None):
+def load(handle, tags=None, options=None, bypass_cert=False):
```

Comment:
At this time it would not be ideal to add more optional parameters to this method. It would complicate future changes. 

We are in a discussion around introducing proper hub.Options object for passing configuration such as this one or caching option in the API, but this change would require a proper design review with consideration for backwards compatibility and a plan for transition and clean up.

That why I was suggesting earlier just to limit the change to behavior just controlled via an environment variable.

Reply:
Okay, I will take these changes back and use an environment variable.